pytorch_lightning/utilities/apply_func.py

```diff
@@ -70,40 +76,93 @@ def apply_to_collection(
         dtype: the given function will be applied to all elements of this dtype
         function: the function to apply
         *args: positional arguments (will be forwarded to calls of ``function``)
-        wrong_dtype: the given function won't be applied if this type is specified and the given collections is of
-            the :attr:`wrong_type` even if it is of type :attr`dtype`
+        wrong_dtype: the given function won't be applied if this type is specified and the given collections
+            is of the :attr:`wrong_type` even if it is of type :attr`dtype`
```

Comment:
```suggestion
            is of the :attr:`wrong_dtype` even if it is of type :attr:`dtype`
```

Reply:
what does attr even do here? afaik it's only for attributes, not function arguments.